lib/collective-sections.js

```diff
@@ -44,15 +47,21 @@ const DEFAULT_SECTIONS = {
     Sections.CONTRIBUTORS,
     Sections.RECURRING_CONTRIBUTIONS,
     Sections.ABOUT,
+    // v2 only
+    Sections.TOP_FINANCIAL_CONTRIBUTORS,
+    Sections.EVENTS,
   ],
   [CollectiveType.FUND]: [
     Sections.CONTRIBUTE,
-    Sections.UPDATES,
     Sections.PROJECTS,
     Sections.BUDGET,
     Sections.CONTRIBUTIONS,
     Sections.CONTRIBUTORS,
     Sections.ABOUT,
+    // v1 but to be removed in v2
+    Sections.UPDATES,
```

Comment:
I don't think this one will be removed in v2, will it?

Reply:
According to the spreadsheet, FUNDS don't have UPDATES https://docs.google.com/spreadsheets/d/15ppKaZJCXBjvY7-AjjCj3w5D-4ebLQdEowynJksgDXE/edit#gid=0
<img width="432" alt="Screenshot 2020-12-17 at 10 29 20" src="https://user-images.githubusercontent.com/37520401/102476218-c023ce00-4052-11eb-826e-0c01089860c2.png">